src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-card flat id="upload-memorandum">
+
+    <!-- Summary Section -->
+    <div id="upload-memorandum-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="upload-memorandum-summary-header" >
+        <v-icon color="#38598A">mdi-text-box-multiple</v-icon>
+        <label class="upload-memorandum-title"><strong>Memorandum</strong></label>
```

Comment:
Geek alert.
This should actually be "bold" not "strong".
https://www.geeksforgeeks.org/difference-between-strong-and-bold-tag-in-html/
End alert.

Reply:
ahh! Will fix :) 
Perhaps i should just be using the `font-weight-bold` helper class anywho?

Reply:
I was trying to resolve (for myself) the difference between `<b>` and `font-weight: 700`, and best as I can tell, they're equivalent. (Bold assumes 700.) So up to you.

Reply:
ok right on!

Reply:
Updated in next commit 👍